mobile_app/test/test_trigger_build.py

```diff
@@ -0,0 +1,139 @@
+"""
+Tests for trigger app build script
+"""
+
+import git
+import json
+from mock import Mock, patch
+import os
+import tempfile
+from unittest import TestCase
+
+from mobile_app import trigger_build
+
+TEST_BRANCH = "test-branch"
+
+
+class Aborted(Exception):
+    """
+    Helper exception for indicating system exit instead of
+    actually exiting the process
+    """
+    pass
+
+
+def _trigger_build(repo_path, environ):
```

Comment:
In terms of convention and readability, I would suggest a few options:
1. Leave them as is, but move them below the Test Class (per Clean Code).
2. Make them static methods in the Test class - so they won't have the `self` parameter.
3. Make them instance methods in the Test class - and ignore the `self` param.